### DISCUSSION TITLE
How can I understand github graph

### DISCUSSION THREAD

**Discussion by Akash-Gupta-git** [sanjay-kv/Open-source-Practice#1190] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/85242e85-4174-4e0f-8a6b-e71e6ff8725c)

This graph is very creative for me I want to understand it properly

This will help me for commit, error, pull request,

**Comment by muhammadadilnaeem** [comment]
Seems Interesting.

**Reply by Akash-Gupta-git** [reply, marked as answer]
--hmm--

**Comment by codeftsaikat** [comment]
Squares and Colors:

Each square = one day.
Darker squares = more activity (commits, pull requests, etc.).
Hover for Details:

Hover over a square to see contributions for that day.
Rows and Columns:

Rows = days (Sunday to Saturday).
Columns = weeks.
Activity Summary:

Below the graph, see total contributions split by type (commits, issues, etc.).
Tip: A darker graph = consistent contributions = good impression!
